<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Zi Ree</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout">
    <property name="margin">
     <number>1</number>
    </property>
    <item row="0" column="0" rowspan="2">
     <widget class="QMdiArea" name="mdiArea">
      <property name="viewMode">
       <enum>QMdiArea::TabbedView</enum>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>868</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="fileNewAction"/>
    <addaction name="fileOpenAction"/>
    <addaction name="fileAddAction"/>
    <addaction name="separator"/>
    <addaction name="fileSaveAction"/>
    <addaction name="fileSaveAsAction"/>
    <addaction name="fileExportForSecondLifeAction"/>
    <addaction name="separator"/>
    <addaction name="fileLoadPropsAction"/>
    <addaction name="fileSavePropsAction"/>
    <addaction name="separator"/>
    <addaction name="fileCloseAction"/>
    <addaction name="fileQuitAction"/>
   </widget>
   <widget class="QMenu" name="menuEdt">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="editUndoAction"/>
    <addaction name="editRedoAction"/>
    <addaction name="separator"/>
    <addaction name="editCutAction"/>
    <addaction name="editCopyAction"/>
    <addaction name="editPasteAction"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="optionsSkeletonAction"/>
    <addaction name="optionsJointLimitsAction"/>
    <addaction name="optionsLoopAction"/>
    <addaction name="optionsProtectFirstFrameAction"/>
    <addaction name="optionsShowTimelineAction"/>
    <addaction name="optionsConfigureQAvimatorAction"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="helpContentsAction"/>
    <addaction name="helpIndexAction"/>
    <addaction name="separator"/>
    <addaction name="helpAboutAction"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="toolsOptimizeBVHAction"/>
    <addaction name="toolsMirrorAction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdt"/>
   <addaction name="menuTools"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolTip">
    <string notr="true"/>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="fileNewAction"/>
   <addaction name="fileOpenAction"/>
   <addaction name="fileAddAction"/>
   <addaction name="fileCloseAction"/>
   <addaction name="separator"/>
   <addaction name="fileSaveAction"/>
   <addaction name="fileSaveAsAction"/>
   <addaction name="separator"/>
   <addaction name="editCutAction"/>
   <addaction name="editCopyAction"/>
   <addaction name="editPasteAction"/>
   <addaction name="separator"/>
   <addaction name="editUndoAction"/>
   <addaction name="editRedoAction"/>
   <addaction name="separator"/>
   <addaction name="resetCameraAction"/>
  </widget>
  <action name="fileNewAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/filenew.png</normaloff>:/icons/icons/filenew.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="fileOpenAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/fileopen.png</normaloff>:/icons/icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Open ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="fileAddAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/fileadd.png</normaloff>:/icons/icons/fileadd.png</iconset>
   </property>
   <property name="text">
    <string>Add...</string>
   </property>
  </action>
  <action name="fileSaveAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/filesave.png</normaloff>:/icons/icons/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="fileSaveAsAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/filesaveas.png</normaloff>:/icons/icons/filesaveas.png</iconset>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="fileLoadPropsAction">
   <property name="text">
    <string>Load Props...</string>
   </property>
  </action>
  <action name="fileSavePropsAction">
   <property name="text">
    <string>Save Props...</string>
   </property>
  </action>
  <action name="fileQuitAction">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="editUndoAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/undo.png</normaloff>:/icons/icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="editRedoAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/redo.png</normaloff>:/icons/icons/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="editCutAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/editcut.png</normaloff>:/icons/icons/editcut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="editCopyAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/editcopy.png</normaloff>:/icons/icons/editcopy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="editPasteAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/editpaste.png</normaloff>:/icons/icons/editpaste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="toolsOptimizeBVHAction">
   <property name="text">
    <string>Optimize BVH</string>
   </property>
  </action>
  <action name="optionsSkeletonAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Skeleton</string>
   </property>
  </action>
  <action name="optionsJointLimitsAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Joint Limits</string>
   </property>
  </action>
  <action name="optionsLoopAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Loop</string>
   </property>
  </action>
  <action name="optionsProtectFirstFrameAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Protect First Frame</string>
   </property>
  </action>
  <action name="optionsShowTimelineAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Timeline</string>
   </property>
  </action>
  <action name="optionsConfigureQAvimatorAction">
   <property name="text">
    <string>Configure QAvimator...</string>
   </property>
  </action>
  <action name="helpContentsAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Contents...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+?, Backspace, Backspace, Del</string>
   </property>
  </action>
  <action name="helpIndexAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Index...</string>
   </property>
  </action>
  <action name="helpAboutAction">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="resetCameraAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/resetcamera.png</normaloff>:/icons/icons/resetcamera.png</iconset>
   </property>
   <property name="text">
    <string>Reset Camera</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="fileExportForSecondLifeAction">
   <property name="text">
    <string>Export For Second Life</string>
   </property>
  </action>
  <action name="toolsMirrorAction">
   <property name="text">
    <string>Mirror</string>
   </property>
   <property name="toolTip">
    <string>Mirror Animation or Track</string>
   </property>
  </action>
  <action name="fileCloseAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/delete.png</normaloff>:/icons/icons/delete.png</iconset>
   </property>
   <property name="text">
    <string>Close file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
